code-studio/src/js/components/share_dialog.jsx

```diff
@@ -1,5 +1,5 @@
 var ShareDialogBody = require('./share_dialog_body.jsx');
-var Dialog = require('./dialog.jsx');
+var Dialog = require('@cdo/apps/templates/Dialog.jsx');
```

Comment:
I think so.

* code-studio/dashboard already knows about apps - it depends on it, because it acts as a wrapper for it.  See initApp.js and all of the dashboard models that know about app-specific options.
* apps does not, and should not, know anything about code-studio/dashboard, except for its ability to interact with our REST APIs.